src/features/surveys/components/SurveySubmissionsList.tsx

```diff
@@ -1,25 +1,23 @@
-import { useQuery } from 'react-query';
-import { useRouter } from 'next/router';
-import { FC, useMemo } from 'react';
-
 import { Box } from '@mui/system';
-import { Link } from '@mui/material';
-import {
-  DataGridPro,
-  GridCellParams,
-  GridRenderCellParams,
-  useGridApiContext,
-} from '@mui/x-data-grid-pro';
-
 import getPeopleSearchResults from 'utils/fetching/getPeopleSearchResults';
+import { Link } from '@mui/material';
```

Comment:
Now you've made _more_ changes to this file, which is unrelated to this PR. It's ok, but it does confuse the reviewer and future readers of the code when changes are included in commits when they are unrelated to the commit.

Reply:
Got it, I´ll try to keep that in mind in the future, thanks for the review!